stream-chat-android-ui-components/src/main/kotlin/io/getstream/chat/android/ui/messages/reactions/UserReactionsView.kt

```diff
@@ -0,0 +1,75 @@
+package io.getstream.chat.android.ui.messages.reactions
+
+import android.content.Context
+import android.util.AttributeSet
+import android.widget.FrameLayout
+import androidx.recyclerview.widget.GridLayoutManager
+import io.getstream.chat.android.client.models.Message
+import io.getstream.chat.android.ui.R
+import io.getstream.chat.android.ui.databinding.StreamUserReactionsViewBinding
+import io.getstream.chat.android.ui.utils.extensions.inflater
+
+public class UserReactionsView : FrameLayout {
+
+    private val binding: StreamUserReactionsViewBinding =
+        StreamUserReactionsViewBinding.inflate(context.inflater, this, true)
+
+    private var reactionClickListener: ReactionsView.ReactionClickListener? = null
+    private val userReactionsAdapter: UserReactionAdapter = UserReactionAdapter {
+        reactionClickListener?.onReactionClick(it)
+    }
+
+    public constructor(context: Context) : super(context) {
+        init()
+    }
+
+    public constructor(context: Context, attrs: AttributeSet?) : super(context, attrs) {
+        init()
+    }
+
+    public constructor(context: Context, attrs: AttributeSet?, defStyleAttr: Int) : super(
+        context,
+        attrs,
+        defStyleAttr
+    ) {
+        init()
+    }
+
+    public fun setMessage(message: Message) {
+        bindTitle(message)
+        bindReactionList(message)
+    }
+
+    public fun setReactionClickListener(reactionClickListener: ReactionsView.ReactionClickListener) {
```

Comment:
I'd allow nullable parameters here, just so you can clear the listener easily - the property already allows for it